ballerina/context.bal

```diff
@@ -22,87 +22,65 @@ import ballerina/lang.value;
 
 # The GraphQL context object used to pass the meta information between resolvers.
 public isolated class Context {
-    private final map<value:Cloneable|isolated object {}> attributes = {};
     private final ErrorDetail[] errors = [];
     private Engine? engine = ();
-    private int nextInterceptor = 0;
     private boolean hasFileInfo = false; // This field value changed by setFileInfo method
-    private map<dataloader:DataLoader> idDataLoaderMap = {}; // Provides mapping between user defined id and DataLoader
-    private map<Placeholder> uuidPlaceholderMap = {};
-    private Placeholder[] unResolvedPlaceholders = [];
-    private boolean containPlaceholders = false;
-    private int unResolvedPlaceholderCount = 0; // Tracks the number of Placeholders needs to be resolved
-    private int unResolvedPlaceholderNodeCount = 0; // Tracks the number of nodes to be replaced in the value tree
+
+    public isolated function init() {
+        self.initializeContext();
+    }
+
+    isolated function initializeContext() = @java:Method {
+        'class: "io.ballerina.stdlib.graphql.runtime.engine.Context"
+    } external;
 
     # Sets a given value for a given key in the GraphQL context.
     #
     # + key - The key for the value to be set
     # + value - Value to be set
     public isolated function set(string 'key, value:Cloneable|isolated object {} value) {
-        lock {
-            if value is value:Cloneable {
-                self.attributes['key] = value.clone();
-            } else {
-                self.attributes['key] = value;
-            }
-        }
+        self.setAttribute('key, value is value:Cloneable ? value.clone() : value);
     }
 
     # Retrieves a value using the given key from the GraphQL context.
     #
```

Comment:
Shall we include a sample code here, similar to what we have in langlib functions? Something like: (Please check the syntax, it might be wrong)

```suggestion
# ```
# string userId = check context.get("userId").ensureType();
# ```
```